vendor/knative.dev/reconciler-test/pkg/environment/magic.go

```diff
@@ -183,7 +183,7 @@ func WithEmitter(emitter milestone.Emitter) EnvOpts {
 }
 
 func (mr *MagicGlobalEnvironment) Environment(opts ...EnvOpts) (context.Context, Environment) {
-	opts = append([]EnvOpts{inNamespace()}, opts...)
+	opts = append(opts, inNamespace())
```

Comment:
@creydr  these are the latest changes, from 1.10 
I saw your upstream PR does same.

We want that here, on 1.10 too?

Reply:
> We want that here, on 1.10 too?
I would assume so